### DISCUSSION TITLE
brew update - how can I see only the updates that will be installed?

### DISCUSSION THREAD

**Discussion by gcstang** [Homebrew/discussions#3067] [Everyday usage]
I'm attempting to capture the output of `brew update` and then based on something being installed run a permission fix
`sudo chown -R $(whoami) /usr/local/bin /usr/local/lib`

I can capture the `outdated formulae installed` line but that just provides a number and not what will be installed.

I currently do this to capture the output but it doesn't keep the green check marks next to what will be installed when I print it.
`output=$(brew update 2>&1 )`

I've tried printing it with both of these but no luck.
`echo -e "$output"`
`printf "$output"`

Is there a command or flag I can run with `brew update` or after to just output the new updates that will be installed with `brew upgrade` ?

**Comment by carlocab** [comment]
Does `brew upgrade --dry-run` work for you?

**Reply by gcstang** [reply]
That worked, thank you.
